docs/plugins/videoplayer.md

```diff
@@ -1,31 +1,32 @@
-# VideoPlayer plugin
+# VideoPlayer
 
-A common feature of TV apps is to play videos. The VideoPlayer plugin offers a convenient interface
-to interact with the video player of the STB. You can use it to open and play / pause videos.
-But it also offers APIs to control the size of the video player, for example.
+A common feature of TV Apps is to play videos.
 
-The VideoPlayer plugin has a built-in integration with the Metrics plugin. It automatically sends statistics for
-various media events (for example: canplay, play, pause, seeking, seeked).
+The *VideoPlayer* plugin offers a convenient interface
+for interacting with the video player of the STB. You can use it to open and play / pause videos. Additionally, it provides APIs that you can use to, for example, control the size of the video player.
 
-Although it's possible to implement a fully custom video playback solution, the use of the VideoPlayer plugin from
+The VideoPlayer plugin has a built-in integration with the [Metrics](metrics.md) plugin. It automatically sends statistics for
+various [media events](#events) (for example: canplay, play, pause, seeking, seeked).
+
+> Although it is possible to implement a fully custom video playback solution, the use of the VideoPlayer plugin from
 the SDK is highly recommended.
 
 ## Usage
 
-In the Lightning components that require video playback capabilities (i.e., a Player component), you can import the
-VideoPlayer plugin from the Lightning SDK.
+In the Lightning components that require video playback capabilities (i.e., Player components), you can import the
```

Comment:
I think I would keep this singular. An App usually has 1 Player component.

Reply:
Corrected.